util/feedback.py

```diff
@@ -22,4 +23,9 @@ async def delete_feedback(event_id, feedback_id, db):
         raise HTTPException(status_code=404,
                             detail="Feedback ID not found in the provided event")
                             
-    column.update_one({"_id": event_id}, {"$set": found_event})
\ No newline at end of file
+    column.update_one({"_id": event_id}, {"$set": found_event})
+
+    def add_upvote_to_feedback(feedback_id, db):
+        #addupvote
+        collection = db[DB_NAME]["feedback"]
+        collection.update_one({"_id": feedback_id}, {"$inc": {"upvotes": 1}})
```

Comment:
this whole code block is indented (probably accident and the linter will catch it)